### PR TITLE
ERROR(Named arguments) -- softmax_cross_entropy_with_logits

### PR DESCRIPTION
**Pull request by aashishrn**
In recent tensorflow versions(i am using 1.140) "softmax_cross_entropy_with_logits" doesn't take arguments without names. Hence this gave an error. It can be fixed by naming arguments "logits" and "labels" as i did in the changes proposed.

### CODE REVIEW

**Comment by ucalyptus** [comment]
@aashishrn  That fraudster doesn't care!
